anomalib/deploy/optimize.py

```diff
@@ -46,6 +46,7 @@ def export_convert(
     input_size: Union[List[int], Tuple[int, int]],
     onnx_path: Union[str, Path],
     export_path: Union[str, Path],
```

Comment:
Would it be possible to have a single path? Like remove `onnx_path` and keep `export_path` only?

Reply:
I think the idea in having two paths is for all of the OpenVINO files to be stored in a single directory while the ONNX file is on the same directory level as `weights`. @ashwinvaidya17 I see you wrote this function is that a correct understanding?

I like having two directories one for ONNX and one for OpenVINO, but I am open to refactoring if needed.

Reply:
if we want to have two path variables for onnx and openvino, then the name of `export_path` would be `openvino_path`. This is because `onnx` is also part of `export` 

> Maybe we can put all of them in a single folder and then see what feedback we get from the community.

Maybe we could have `export_path` as `Optional[Union[str, Path]] = None`. If None, it could be the same directory as the original weight file. Otherwise, we could save it to the `export_path`. thoughts?

Reply:
Okay I tried a stab at it in: [a4b0604](https://github.com/openvinotoolkit/anomalib/pull/509/commits/a4b060460905a5e4ce7340f128a4ee8717c4cd66)

Let me know if this works. We still internal to `export_convert` function has `onnx_path` but nothing the user has to set from the config

Reply:
Thanks, only a single comment

Reply:
Fixed in [019460d](https://github.com/openvinotoolkit/anomalib/pull/509/commits/019460da3881f777ff5d3f2f9bc7495c641508af)